docs/faq/overview.md

```diff
@@ -0,0 +1,17 @@
+# F.A.Q
+
+## What is the Hedgehog Fabric
```

Comment:
```suggestion
## What is the Hedgehog Fabric?

```

Additional questions: “What is the Open Network Fabric? How does it differ from the Hedgehog Fabric?” :innocent: